gtsfm/data_association/point3d_initializer.py

```diff
@@ -176,15 +174,16 @@ def triangulate(self, track_2d: SfmTrack2d) -> Tuple[Optional[SfmTrack], Optiona
         camera_track, measurement_track = self.extract_measurements(inlier_track)
         try:
             triangulated_pt = gtsam.triangulatePoint3(
-                camera_track,
-                measurement_track,
-                rank_tol=SVD_DLT_RANK_TOL,
-                optimize=True,
+                camera_track, measurement_track, rank_tol=SVD_DLT_RANK_TOL, optimize=True,
             )
         except RuntimeError:
             is_cheirality_failure = True
             return None, None, is_cheirality_failure
 
+        # check for small baseline case
+        if self.is_small_triangulation_angle(triangulated_pt, inlier_idxs):
```

Comment:
Ayush, could you show the full metrics on master vs. on this branch? and a qualitative image before and after?

Reply:
I have added the metrics on the PR. Also, by default this check will remain off.